Modules/Module_Processing/xASL_init_DataLoading.m

```diff
@@ -63,6 +77,9 @@
         % Optionally, developers have set this parameter to force reading subjects from /rawdata
         warning(['rawdata folder missing: ' x.dir.RawData]);
         error('rawdata did not exist, if you want to read subjects from /derivatives/ExploreASL, set x.opts.bReadRawdata to false in dataPar.json');
+    elseif x.opts.bReadRawdata && isequal(x.opts.bProcess(:), [0; 0; 1])
```

Comment:
This is not true right?
If the user specifically set bReadRawdata to true, we don't change it

Reply:
I think so. Did you check my last commit?

Reply:
Yes. It's all good and you can approve.